plugins/balloontoolbar/plugin.js

```diff
@@ -723,6 +729,24 @@
 
 				CKEDITOR.ui.balloonPanel.prototype.attach.call( this, element, options );
 			};
+
+			/**
+			 * Method deregister all focusable items from balloontoolbar.
+			 *
+			 * @private
+			 * @since 4.8.1
+			 * @member CKEDITOR.ui.balloonToolbarView
+			 */
+			CKEDITOR.ui.balloonToolbarView.prototype.deregisterFocusableItems = function() {
+				var focusables = this.focusables;
+				for ( var id in focusables ) {
+					if ( focusables.hasOwnProperty( id ) ) {
+						if ( focusables[ id ].getId() && focusables[ id ].getName() === 'a' ) {
```

Comment:
In https://github.com/ckeditor/ckeditor-dev/pull/1327#discussion_r157462771 I have asked for slightly different thing:

> We have to remove focusables that are within content part.
>
> To do that we could simply iterate over it and test each focusable if it is a child of content part and remove it if it actually is (do this check for each focusable). And we won't have to worry about logic duplication.

This could solved simply with code like that:

```js
var focusables = this.focusables;
for ( var id in focusables ) {
	if ( this.parts.content.contains( focusables[ id ] ) ) {
		this.deregisterFocusable( focusables[ id ] );
	}
}
```